lib/apidoco/folder_parser.rb

```diff
@@ -21,7 +21,7 @@ def basename
     end
 
     def children
-      directory.children.map do |child|
+      directory.children.sort.map do |child|
```

Comment:
I am not sure which attribute will get used for sorting. @alfie-max Can you more details?

Reply:
alphabetically based on the directory names would be my first thought

Reply:
Please work on https://github.com/72pulses/apidoco/blob/360cae7669a575b19b058ffc769658a5faee222f/lib/apidoco/file_parser.rb#L27 method instead. Thanks!

Reply:
that's the file parser. 

i get that we can specify the sort order for each `.json` file
But what I'm talking is about the sort order of subdirectories

Here's my directory structure.. 
![screenshot 2018-12-27 at 11 25 03 pm](https://user-images.githubusercontent.com/8247590/50489606-ad133000-0a2e-11e9-8a1d-c537083d3bb1.png)

So with this PR, i'm aiming to alphabetically sort `authentication` `companies` `products`

By default `products` comes first where as i'd expect `authetication` would have